task4/Task4/Query.cs

```diff
@@ -8,8 +8,11 @@ public class Query
             "query 2",
             "query 3",
             "query 4",
-            "query 5",
-            "query 6",
+            "SELECT COUNT(*) FROM `Employees` WHERE `City`='@CityName';",
+            "SELECT MAX((YEAR(CURRENT_TIMESTAMP)-YEAR(`BirthDate`))) AS MaxAge, " +
```

Comment:
How about replacing this line with `(YEAR(CURRENT_TIMESTAMP)-MIN(YEAR(BirthDate)))` ?
Will it work?